src/v2/Apps/Collect/Routes/Collection/index.tsx

```diff
@@ -69,6 +69,10 @@ export const CollectionApp: React.FC<CollectionAppProps> = props => {
       <Meta property="og:url" content={collectionHref} />
       <Meta property="og:image" content={socialImage} />
       <Meta property="og:description" content={metadataDescription} />
+      <Meta property="og:title" content={title} />
+      <Meta property="og:type" content="website" />
+      <Meta property="og:site_name" content="Artsy" />
+      <Meta property="twitter:card" content="summary_large_image" />
```

Comment:
@mdole - note that we now have a `MetaTags` component. That might be helpful here.

Reply:
https://github.com/artsy/force/blob/7a216215d6c9d0fe3fee63dcc8f6a1fc376dcd19/src/v2/Apps/ArtistSeries/Components/ArtistSeriesMeta.tsx#L24-L28

Reply:
oooo that's awesome, thanks for the pointer! Updated in https://github.com/artsy/force/pull/8084